src/components/InventoryV2/InventoryListByBatch.tsx

```diff
@@ -0,0 +1,160 @@
+  { key: 'batchNumber', name: strings.BATCH_NUMBER, type: 'string', tooltipTitle: strings.TOOLTIP_BATCH_NUMBER },
+  {
+    key: 'species_scientificName_noLink',
+    name: strings.SPECIES,
+    type: 'string',
+    tooltipTitle: strings.TOOLTIP_SCIENTIFIC_NAME,
+  },
+  {
+    key: 'species_commonName',
+    name: strings.COMMON_NAME,
+    type: 'string',
+  },
+  {
+    key: 'facility_name_noLink',
+    name: strings.NURSERY,
+    type: 'string',
+  },
+  {
+    key: 'germinatingQuantity',
+    name: strings.GERMINATING,
+    type: 'string',
+    tooltipTitle: strings.TOOLTIP_GERMINATING_QUANTITY,
+  },
+  {
+    key: 'notReadyQuantity',
+    name: strings.NOT_READY,
+    type: 'string',
+    tooltipTitle: strings.TOOLTIP_NOT_READY_QUANTITY,
+  },
+  { key: 'readyQuantity', name: strings.READY, type: 'string', tooltipTitle: strings.TOOLTIP_READY_QUANTITY },
+  { key: 'totalQuantity', name: strings.TOTAL, type: 'string', tooltipTitle: strings.TOOLTIP_TOTAL_QUANTITY },
+];
+
+export default function InventoryListByBatch() {
+  const theme = useTheme();
+  const classes = useStyles();
+  const { selectedOrganization } = useOrganization();
+  const [searchResults, setSearchResults] = useState<SearchResponseElement[] | null>();
+  const [unfilteredInventory, setUnfilteredInventory] = useState<SearchResponseElement[] | null>(null);
+  const [temporalSearchValue, setTemporalSearchValue] = useState('');
+  const debouncedSearchTerm = useDebounce(temporalSearchValue, 250);
+
+  const [searchSortOrder, setSearchSortOrder] = useState<SearchSortOrder | undefined>({
+    field: 'batchNumber',
+    direction: 'Ascending',
+  });
+  const [filters, setFilters] = useForm<InventoryFiltersType>({});
+
+  const { user } = useUser();
+  const numberFormatter = useNumberFormatter();
+  const numericFormatter = useMemo(() => numberFormatter(user?.locale), [user?.locale, numberFormatter]);
+
+  const onSearchSortOrder = (order: SearchSortOrder) => {
+    const isClientSorted = BE_SORTED_FIELDS.indexOf(order.field) === -1;
+    setSearchSortOrder(isClientSorted ? undefined : order);
+  };
+
+  const onApplyFilters = useCallback(async () => {
+    const requestId = Math.random().toString();
+    setRequestId('searchInventory', requestId);
+    const apiSearchResults = await NurseryBatchService.getAllBatches(
+      selectedOrganization.id,
+      searchSortOrder,
+      filters.facilityIds,
+      debouncedSearchTerm
+    );
+
+    let updatedResult: InventoryResultWithBatchNumber[] | undefined = [];
+
+    // format results
+    updatedResult = apiSearchResults?.map((uR) => {
+      const resultTyped = uR as BatchInventoryResult;
+      return {
+        ...resultTyped,
+        batchId: resultTyped.id,
+        species_scientificName_noLink: resultTyped.species_scientificName,
+        facility_name_noLink: resultTyped.facility_name,
+        germinatingQuantity: numericFormatter.format(resultTyped['germinatingQuantity(raw)']),
```

Comment:
I was under the impression that accessing `germinatingQuantity`, without the `(raw)`, part is already localized... is that right? If that's true, would it be OK to just use that?

Reply:
yes, good catch

Reply:
Good point, I'll change that. Thanks